tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -99,4 +103,115 @@ def test_max_login_failure(monkeypatch):
+    job = JobFactory(submission_id=1, job_status=sess.query(JobStatus).filter_by(name='finished').one(),
+                     job_type=sess.query(JobType).filter_by(name='csv_record_validation').one(),
+                     file_type=sess.query(FileType).filter_by(name='award').one())
+    add_models(database, [user, sub, job])
+
+    json_response = ah.list_submissions(None, None, None)
+    assert json.loads(json_response.get_data().decode("utf-8"))['total'] == 1
+    assert json.loads(json_response.get_data().decode("utf-8"))['submissions'][0]['status'] == "validation_successful"
+    delete_models(database, [user, sub, job])
+
+    sess = database.session
+    user = UserFactory(user_id=1, cgac_code='cgac')
+    sub = SubmissionFactory(user_id=1, submission_id=1, cgac_code='cgac')
+    job = JobFactory(submission_id=1, job_status=sess.query(JobStatus).filter_by(name='running').one(),
+                     job_type=sess.query(JobType).filter_by(name='csv_record_validation').one(),
+                     file_type=sess.query(FileType).filter_by(name='award').one())
+    add_models(database, [user, sub, job])
+
+    json_response = ah.list_submissions(None, None, None)
+    assert json.loads(json_response.get_data().decode("utf-8"))['total'] == 1
+    assert json.loads(json_response.get_data().decode("utf-8"))['submissions'][0]['status'] == "running"
+    delete_models(database, [user, sub, job])
+
+    sess = database.session
+    user = UserFactory(user_id=1, cgac_code='cgac')
+    sub = SubmissionFactory(user_id=1, submission_id=1, cgac_code='cgac')
+    job = JobFactory(submission_id=1, job_status=sess.query(JobStatus).filter_by(name='waiting').one(),
+                     job_type=sess.query(JobType).filter_by(name='csv_record_validation').one(),
+                     file_type=sess.query(FileType).filter_by(name='award').one())
+    add_models(database, [user, sub, job])
+
+    json_response = ah.list_submissions(None, None, None)
+    assert json.loads(json_response.get_data().decode("utf-8"))['total'] == 1
+    assert json.loads(json_response.get_data().decode("utf-8"))['submissions'][0]['status'] == "waiting"
+    delete_models(database, [user, sub, job])
+
+    sess = database.session
+    user = UserFactory(user_id=1, cgac_code='cgac')
+    sub = SubmissionFactory(user_id=1, submission_id=1, cgac_code='cgac')
+    job = JobFactory(submission_id=1, job_status=sess.query(JobStatus).filter_by(name='ready').one(),
+                     job_type=sess.query(JobType).filter_by(name='csv_record_validation').one(),
+                     file_type=sess.query(FileType).filter_by(name='award').one())
+    add_models(database, [user, sub, job])
+
+    json_response = ah.list_submissions(None, None, None)
+    assert json.loads(json_response.get_data().decode("utf-8"))['total'] == 1
+    assert json.loads(json_response.get_data().decode("utf-8"))['submissions'][0]['status'] == "ready"
+    delete_models(database, [user, sub, job])
```

Comment:
I added those to specifically test that the correct status comes back based on what the related jobs' statuses were. I wanted to check every possible status to make sure we get back what's expected when a job has a specific status.